### ISSUE TITLE
unknown library "Function" autoadd.

### ISSUE THREAD

**Issue by houd1ni** [timocov/dts-bundle-generator#101]
Hi! I've just updated deps for https://github.com/houd1ni/lafetch/

Node 13.2.1 (Current stable).
This lib: 3.3.1.
TypeScript: 3.7.2

I get:
```
Compiling input files...
Processing src/main.ts
Adding import with name "Curry" for library "Function"
Writing src/main.ts -> dist/bundle.d.ts
Checking generated files...
dist/bundle.d.ts(1,23): error TS2307: Cannot find module 'Function'.
```
And have it as a first line in my bundle.ts:
```
import { Curry } from 'Function';
```
Also, I have no Curry used there in sources. Only ramda's curry.

After downgrading the lib to 3.1.0 and @types/ramda back to 0.26.14 I get this:
```
error TS6053: File 'src/main.d.ts' not found.
```

And only when I've downgraded typescript to 3.5.2, I finally have successful build.

**Comment by timocov** [comment]
Hey @houd1ni, sorry for late reply.

I just clone your repo, `npm install` and run `npm run gentypes` and no error happened. After that I run `npm install -D typescript@3.7.3 dts-bundle-generator@3.3.1` to update to the latest version packages and run `npm run gentypes` again and the result is the same - no errors. How to reproduce the errors you've faced?

**Comment by houd1ni** [comment]
@timocov  Try to upgrade @types/ramda to the latest: 0.26.36.
The problem shows up during `Checking generated files...`.
Checks in IDE(vscode) and during compilation looks OK.

**Comment by timocov** [comment]
Fixed in #103
